src/CoreBundle/private/library/classes/TCMSMessageManager/TCMSMessageManager.class.php

```diff
@@ -223,11 +223,12 @@ protected function AddBackendMessage($sConsumerName, $sMessageCode, $aMessageCod
      * the end of the list.
      *
      * @param string $sConsumerName
-     * @param bool   $bRemove
+     * @param bool $bRemove
+     * @param bool $includeGlobalConsumer
      *
      * @return TIterator
      */
-    public function ConsumeMessages($sConsumerName, $bRemove = true)
+    public function ConsumeMessages($sConsumerName, $bRemove = true, bool $includeGlobalConsumer = true)
```

Comment:
"include global consumer name messages" is what the new name means. (it's not the consumer that is included)

I would call it "includeGlobal" for short.

Reply:
Done